src/pages/ShoppingCart.page.js

```diff
@@ -1,27 +1,17 @@
-import { BaseSwagLabPage } from './BaseSwagLab.page';
+import { BasePage } from './Base.page';
 
-export class ShoppingCartPage extends BaseSwagLabPage {
+export class ShoppingCartPage extends BasePage {
     url = '/cart.html';
 
-    cartItemSelector = '.cart_item';
+    headerTitle = this.page.getByTestId('title');
 
-    removeItemSelector = '[id^="remove"]';
+    inventoryItems = this.page.getByTestId('inventory-item');
 
-    headerTitle = this.page.locator('.title');
+    removeButtonFirstProduct = this.page.getByTestId(
+        'remove-sauce-labs-backpack',
```

Comment:
This locator is very unstable. When the products list will be changed U'll need to fix it everytime